packages/suite/src/reducers/suite/messageSystemReducer.ts

```diff
@@ -54,12 +54,12 @@ const messageSystemReducer = (state: State = initialState, action: Action): Stat
                     ...action.payload.messageSystem,
                 };
             case MESSAGE_SYSTEM.FETCH_CONFIG_SUCCESS:
-                draft.timestamp = Date.now();
```

Comment:
reducers should be pure... https://redux.js.org/understanding/thinking-in-redux/three-principles#changes-are-made-with-pure-functions